models/app.models.js

```diff
@@ -6,12 +6,14 @@ exports.fetchTopics = () => {
     })
 }
 
-exports.fetchArticleById = (id) =>{
+exports.fetchArticleById = (article_id) =>{
+  if (isNaN(article_id)) return Promise.reject({ status: 400, message: 'Not a number, please enter valid id'});
```

Comment:
This could be left to a PSQL error handler - as the query will reject if the article_id isn't an integer :)

Reply:
I fixed this now.